project.Makefile

```diff
@@ -197,9 +206,30 @@ local/mongo_as_unvalidated_nmdc_database.yaml:
 		--selected-collections read_based_taxonomy_analysis_activity_set \
 		--selected-collections read_qc_analysis_activity_set \
 		--selected-collections study_set \
-		--selected-collections data_object_set \
-		--selected-collections field_research_site_set \
-		--skip-collection-check
+		--selected-collections workflow_chain_set \
+		--selected-collections workflow_execution_set \
+		api-access \
+		--client-base-url "https://api-napa.microbiomedata.org" \
```

Comment:
I see the value here is the Napa Runtime API URL.

Reply:
do you have a request or suggestion?

Reply:
I was surprised to see the Napa environment's URL being hard-coded anywhere (other than in re-ID-ing-specific code), since it is a "temporary" environment.

Reply:
No particular suggestion. I'm OK with it being the Napa environment—I just was surprised to see it (given that the _production_ environment was used as a default elsewhere in the diff) and wanted to make sure it was intentional.

Reply:
I guess I should switch the usage example to the production endpoint. I'll do all of this Monday. THanks